docs/ctrlmesh/intro/intro.md

```diff
@@ -0,0 +1,21 @@
+
+# Controller Mesh
+
+KusionStack Controller Mesh is a solution that helps developers manage their controllers/operators better.
+
+## Key Features
+
+1. **Sharding**: Through relevant configurations, Kubernetes single-point deployed operator applications can be flexibly shard deployed.
+2. **Canary upgrade**: Depends on sharding, the controllers can be updated in canary progress instead of one time replace.
```

Comment:
`the controllers can be updated in canary progress instead of one time replace.`
=>
`the controller instances can be updated in canary progress instead of updated in one time.`?